site/blog/airship-update-march-2020.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Airship Update - March 2020
+author: alex-hughes
+date: 2020-03-11T12:00:00.000Z
+category: Airship 2.0
+---
+
+What follows is the Airship Community Update for the month of March, 2020, brought to you by the Airship Technical
+Committee.<!-- more -->
+
+## **SPOTLIGHT ON ....**
+
+- Daylight Saving Time impact on community meetings. If you do not observe Daylight Saving Time please note that meeting
```

Comment:
This is a dependent clause. You need a comma after "observe Daylight Saving Time".

Reply:
done.